edgelet/edgelet-kube/src/runtime.rs

```diff
@@ -176,24 +192,28 @@ impl<T, S> KubeModuleRuntime<T, S> {
             proxy_config_path,
             proxy_config_map_name,
             image_pull_policy,
-            service_account_name,
             workload_uri,
             management_uri,
             device_hub_selector,
         })
     }
+
+    pub fn client(&self) -> Arc<Mutex<RefCell<KubeClient<T, S>>>> {
```

Comment:
I see that you need this from `create.rs`. Can we make this `pub(crate)` at least?